lib/src/rules/prefer_spread_collections.dart

```diff
@@ -99,7 +99,9 @@ class _Visitor extends SimpleAstVisitor<void> {
     CascadeExpression cascade = invocation.thisOrAncestorOfType();
     NodeList<Expression> sections = cascade.cascadeSections;
     Expression target = cascade.target;
-    if (target is! ListLiteral || sections[0] != invocation) {
+    if (target is! ListLiteral ||
+        (target as ListLiteral).isConst ||
```

Comment:
`isConst` return `true` if the literal has a `const` modifier. To catch both that and the case where it's nested inside some larger constant, you need to use `inConstantContext` (which is currently only available on the `Impl` classes).

Reply:
Nice!  I hadn't seen that.  Thank you!